packages/theme/src/components/alert.ts

```diff
@@ -5,87 +5,117 @@ import {
+  w: 5,
+  h: 5,
+}
+
+const baseStyle = {
+  container: baseStyleContainer,
+  title: baseStyleTitle,
+  icon: baseStyleIcon,
+}
+
+const defaultProps = {
+  variant: "subtle",
+}
+
+const variantSubtle = function (props: Record<string, any>) {
+  const { colorScheme: c } = props
+  return {
+    container: { bg: getBg(props) },
+    icon: { color: mode(`${c}.500`, `${c}.200`)(props) },
+  }
+}
+
+const variantLeftAccent = function (props: Record<string, any>) {
+  const { colorScheme: c } = props
+  return {
     container: {
-      px: 4,
-      py: 3,
-    },
-    title: {
-      fontWeight: "bold",
-      lineHeight: "normal",
+      pl: 3,
+      borderLeft: "4px solid",
+      borderColor: mode(`${c}.500`, `${c}.200`)(props),
+      bg: getBg(props),
     },
     icon: {
-      mr: 3,
-      w: 5,
-      h: 5,
-    },
-  },
-  variants: {
-    subtle: function (props) {
-      const { colorScheme: c } = props
-      return {
-        container: { bg: getBg(props) },
-        icon: { color: mode(`${c}.500`, `${c}.200`)(props) },
-      }
+      color: mode(`${c}.500`, `${c}.200`)(props),
     },
+  }
+}
 
-    "left-accent": function (props) {
-      const { colorScheme: c } = props
-      return {
-        container: {
-          pl: 3,
-          borderLeft: "4px solid",
-          borderColor: mode(`${c}.500`, `${c}.200`)(props),
-          bg: getBg(props),
-        },
-        icon: {
-          color: mode(`${c}.500`, `${c}.200`)(props),
-        },
-      }
+const variantTopAccent = function (props: Record<string, any>) {
+  const { colorScheme: c } = props
+  return {
+    container: {
+      pt: 2,
+      borderTop: "4px solid",
+      borderColor: mode(`${c}.500`, `${c}.200`)(props),
+      bg: getBg(props),
     },
-
-    "top-accent": function (props) {
-      const { colorScheme: c } = props
-      return {
-        container: {
-          pt: 2,
-          borderTop: "4px solid",
-          borderColor: mode(`${c}.500`, `${c}.200`)(props),
-          bg: getBg(props),
-        },
-        icon: {
-          color: mode(`${c}.500`, `${c}.200`)(props),
-        },
-      }
+    icon: {
+      color: mode(`${c}.500`, `${c}.200`)(props),
     },
+  }
+}
 
-    solid: function (props) {
-      const { colorScheme: c } = props
-      return {
-        container: {
-          bg: mode(`${c}.500`, `${c}.200`)(props),
-          color: mode(`white`, `gray.900`)(props),
-        },
-      }
+const variantSolid = function (props: Record<string, any>) {
+  const { colorScheme: c } = props
+  return {
+    container: {
+      bg: mode(`${c}.500`, `${c}.200`)(props),
+      color: mode(`white`, `gray.900`)(props),
     },
-  },
+  }
+}
+
+const variants = {
+  subtle: variantSubtle,
+  "left-accent": variantLeftAccent,
+  "top-accent": variantTopAccent,
+  solid: variantSolid,
+}
 
-  defaultProps: {
-    variant: "subtle",
-  },
+const alert = multiStyleConfig({
+  parts,
+  baseStyle,
+  variants,
+  // @ts-ignore
+  defaultProps,
```

Comment:
I think this can be resolved by defining `defaultProps` as `const`:

```ts
const defaultProps = {
  variant: 'subtle',
} as const
```

Reply:
Any consumer defining `defaultProps` as a separate object rather than inline would need to do the same.